src/main/java/com/hackclub/hccore/HCCorePlugin.java

```diff
@@ -155,6 +155,10 @@ private void registerAdvancements() {
                 .getImpossible("ad_astra", millionMiler, "Ad Astra",
                         "Reach outer space and touch the stars", Material.FIREWORK_ROCKET)
                 .setFrame(Advancement.Frame.CHALLENGE);
+        Advancement killGolem = factory
```

Comment:
Can you change it to `kill_iron_golem` and then move it to the line after `kill_wolf`?